integration/kubernetes/nginx.bats

```diff
@@ -22,7 +22,7 @@ setup() {
 }
 
 @test "Verify nginx connectivity between pods" {
-	wait_time=30
+	wait_time=120
 	sleep_time=5
 	cmd="sudo -E kubectl get pods | grep $service_name | grep Running"
```

Comment:
@sboeuf Can you add 
```
sudo -E kubectl describe pod "$busybox_pod" && sudo -E kubectl delete service "$service_name"
``` 
So that we know the reason for failure at least.